docs/upgrade_to_tails_2x.rst

```diff
@@ -0,0 +1,138 @@
+Upgrade Tails from 1.x to 2.x
+=============================
+
+Newer versions of SecureDrop leverage improvements to the Tails OS since the introduction of Tails 2.0 (and subsequent versions). It is critical to upgrade all Tails USBs to the latest version before upgrading SecureDrop. For some admins, this might be complicated, since the upgrade from 1.x to 2.x must be done manually. Please follow this guide to updating the entire system, and special attention to only use the Secure Viewing Station computer so you benefit from its airgap while transfering sensitive data.
```

Comment:
I would rewrite this first paragraph to something like:

If you've been using SecureDrop for a while, you're probably using a version of Tails from the 1.x release series on your Tails USBs for the Admin Workstation, Journalist Workstation, and Secure Viewing Station. Tails has switched over to the 2.x release series, and is no longer supporting the 1.x release series, which means Tails 1.x systems are no longer receiving security updates. As a result, your Tails system will have have a gradually increasing number of security vulnerabilities, which could be used to compromise the security of your SecureDrop instance. As a result, we ***strongly recommend*** upgrading to the latest version of Tails (2.3 at the time of this writing).

In addition to the security concerns, newer versions of SecureDrop leverage improvements to the Tails OS since the introduction of Tails 2.0 (and subsequent versions). It is critical to upgrade all Tails USBs to the latest version before upgrading SecureDrop.

For some admins, this might be complicated, since the upgrade from 1.x to 2.x must be done manually. Please follow this guide to updating the entire system, and special attention to only use the Secure Viewing Station computer so you benefit from its airgap while transfering sensitive data.